parts/linux/cloud-init/artifacts/components.json

```diff
@@ -222,6 +222,7 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "1.13.10",
```

Comment:
are we able to remove older versions for either of these images?

Reply:
Yes, removed now